### ISSUE TITLE
Householding: Add tests (and move to a rule?)

### ISSUE THREAD

**Issue by curtismorales** [HHSIDEAlab/medicaid_eligibility#14]
There should be a set of test for householding logic.

It also probably makes sense to move householding into a ruleset of its own and out of the ApplicationProcessor code.

**Comment by curtismorales** [comment]
@colinxfleming let me know if you have questions.

**Comment by colinxfleming** [comment]
@curtismorales sounds good on tests. Specifically, we're looking for tests of the methods <code>determine_household</code>, <code>calculate_household_size</code>, and <code>calculate_household_income</code> right? Am I leaving anything out? 

w/r/t moving the householding stuff (just those three methods above?) to a rule-style, it would probably make it way more testable for sure. I'll take a whack at it this week.

**Comment by curtismorales** [comment]
Yes to everything. Those three methods. And I think moving it to a rule-style is a good first step.
